.ci/install_rust.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Ant Financial
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+cidir=$(dirname "$0")
+rustarch=$(uname -m)
+release="nightly"
+
+if [ "${rustarch}" == "ppc64le" ]; then
+	rustarch="powerpc64le"
```

Comment:
You could consider adding a `--rust` option to https://github.com/kata-containers/tests/blob/master/.ci/kata-arch.sh

Reply:
fixed